developer-docs-site/docs/tutorials/validator-node/using-docker.md

```diff
@@ -131,6 +135,8 @@ Docker has only been tested on Linux, Windows, and Intel macOS. If you are on M1
     - `waypoint.txt` waypoint for genesis transaction
     - `genesis.blob` genesis binary contains all the info about framework, validatorSet and more.
 
-10. Run docker-compose: `docker-compose up`. (or `docker compose up` depends on your version)
+10. Run docker-compose: `sudo docker-compose up -d`. (or `sudo docker compose up -d` depends on your version)
+    Check the currently running CONTAINER ID `sudo docker ps -a`
+    Check the log's `sudo docker logs -f <CONTAINER ID>`
```

Comment:
sudo is because you haven't set appropriate user permissions and this also makes it more *nix centric. The goal of these docs is to be as platform neutral as possible.

What is the goal for checking the logs?

Reply:
Got it. Running  correctness.